pkg/kudoctl/packages/reader/parser_test.go

```diff
@@ -0,0 +1,70 @@
+package reader
+
+import (
+	"errors"
+	"testing"
+
+	"gotest.tools/assert"
+)
+
+const (
+	validOperator = `
+apiVersion: kudo.dev/v1beta1
+name: "first-operator"
+operatorVersion: "0.1.0"
+`
+	validParams = `
+apiVersion: kudo.dev/v1beta1
+parameters:
+  - name: param
+    default: "value"
+`
+)
+
+func TestParsePackageFile(t *testing.T) {
+	tests := []struct {
+		filePath    string
+		fileContent string
+
+		isOperator bool
+		isParam    bool
+		isTemplate bool
+
+		expectedError error
+	}{
+		{"operator.yaml", validOperator, true, false, false, nil},
```

Comment:
Please use the named field for these structs.  Something like`true, false, false` is hard to read. Goland can even help you with it: `Add keys and delete zero values` in the context help 💡